src/main/webapp/WEB-INF/pages/messages.ftl

```diff
@@ -0,0 +1,166 @@
+<#include "commons.ftl">
```

Comment:
codul nu e formatat corect si foloseste  un amestec de spatii si tab-uri pentru indentare. 
Spring Tool Suite -> Preferences -> General -> Editors -> Text Editors -> bifeaza 'Insert spaces for tabs', apoi intra in fisierele legate de messages si forteaza un nou save, ca sa se aplice modificarea